.circleci/config.yml

```diff
@@ -87,16 +87,6 @@ jobs:
               mv -v tmpfile "<<pipeline.parameters.params_basename>>"
             fi
 
-      - run:
-          name: Set det-version parameter.
```

Comment:
This `det-version` parameter is used elsewhere in `real_config.yml`, e.g. [here](https://github.com/determined-ai/determined/blob/bb315658b9e9fc65c1ea629fcb353e6c96f93555/.circleci/real_config.yml#L768) and [here](https://github.com/determined-ai/determined/blob/bb315658b9e9fc65c1ea629fcb353e6c96f93555/.circleci/real_config.yml#L874), so as it stands, this will break (if there are defaults set, they'll be the empty string). So we either need to generate `det-version` everywhere else in `real_config.yml` where it's used, or come up with something else. It might be possible to run another workflow first that sets `det-version` in `real_config.yml`, but that comes after everything in `config.yml`.

Reply:
My hope was that for each run, we would just have to set the parameter once globally, and then use it wherever we need to.

Reply:
I thought those were regular parameters and not pipeline parameters

Reply:
You're absolutely right, I apologize. I looked too hastily and spoke too soon.